Source/Evolve/WarpXEvolve.cpp

```diff
@@ -405,12 +405,16 @@ WarpX::OneStep_nosub (Real cur_time)
             FillBoundaryE(guard_cells.ng_alloc_EB);
         }
         PushPSATD();
-        FillBoundaryE(guard_cells.ng_alloc_EB);
-        FillBoundaryB(guard_cells.ng_alloc_EB);
 
         if (use_hybrid_QED) {
-            WarpX::Hybrid_QED_Push(dt);
             FillBoundaryE(guard_cells.ng_alloc_EB);
+            FillBoundaryB(guard_cells.ng_alloc_EB);
+            WarpX::Hybrid_QED_Push(dt);
+            FillBoundaryE(guard_cells.ng_afterPushPSATD);
```

Comment:
I don't really know the hybrid qed solver so I left this as this was before (there was only `FillBoundaryE` after `WarpX::Hybrid_QED_Push`).

But looking at the `WarpX::Hybrid_QED_Push` function, I think this is because the hybrid qed solver needs up-to-date values for the E and B field but only modifies the E field.

I think this is a good point, I'll add a one-line comment saying that this function only modifies E.